.github/workflows/sample-application.yml

```
@@ -99,66 +99,39 @@ jobs:
         if: ${{ matrix.platform == 'ios' || matrix.platform == 'macos' }}
         working-directory: samples
         run: |
-          [[ "${{ matrix.platform }}" == "ios" ]] && cd react-native/ios
-          [[ "${{ matrix.platform }}" == "macos" ]] && cd react-native-macos/macos
+          [[ "${{ matrix.platform }}" == "ios" ]] && cd react-native
+          [[ "${{ matrix.platform }}" == "macos" ]] && cd react-native-macos
 
-          [[ "${{ matrix.build-type }}" == "production" ]] && ENABLE_PROD=1 || ENABLE_PROD=0
-          [[ "${{ matrix.rn-architecture }}" == "new" ]] && ENABLE_NEW_ARCH=1 || ENABLE_NEW_ARCH=0
+          [[ "${{ matrix.build-type }}" == "production" ]] && export ENABLE_PROD=1 || export ENABLE_PROD=0
+          [[ "${{ matrix.rn-architecture }}" == "new" ]] && export ENABLE_NEW_ARCH=1 || export ENABLE_NEW_ARCH=0
           [[ "${{ matrix.ios-use-frameworks }}" == "dynamic-frameworks" ]] && export USE_FRAMEWORKS=dynamic
-          echo "ENABLE_PROD=$ENABLE_PROD"
-          echo "ENABLE_NEW_ARCH=$ENABLE_NEW_ARCH"
-          PRODUCTION=$ENABLE_PROD RCT_NEW_ARCH_ENABLED=$ENABLE_NEW_ARCH bundle exec pod install
-          cat Podfile.lock | grep $RN_SENTRY_POD_NAME
+
+          ./scripts/pod-install.sh
 
       - name: Build Android App
         if: ${{ matrix.platform == 'android' }}
-        working-directory: ${{ env.REACT_NATIVE_SAMPLE_PATH }}/android
+        working-directory: ${{ env.REACT_NATIVE_SAMPLE_PATH }}
```

The [Android CI failures](https://github.com/getsentry/sentry-react-native/actions/runs/13392186006/job/37402281182?pr=4561) seem to be related with the working directory
```suggestion
        working-directory: ${{ env.REACT_NATIVE_SAMPLE_PATH }}/android
```

Alternatively we may change the script paths [below](https://github.com/getsentry/sentry-react-native/pull/4561/files#diff-c5263171d4b9aca7af53d70a3ee9f8423c7f4bebd6941e590c5610208f95c8bcR118) to
```
scripts/set-dsn-aos.mjs
scripts/build-android.sh -PreactNativeArchitectures=x86
```

---

Yes, thank you.